File: C++/Maths/Find smallest between two numbers - input values.cpp
#include <iostream>
using namespace std;

int main(){
    int num1, num2;
    cout << "Enter the 1st number: ";
    cin >> num1;
    cout << "Enter the 2nd number: ";
    cin >> num2;

    if(num1 < num2){
        cout << "Smallest number is: " << num1;
    }else if(num2 < num1){
        cout << "Smallest number is: " << num2;
    }else{
        cout << "Smallest number not found. Entered numbers are equal.";
    }
}

File: Python/Maths/Find smallest between two numbers - input values.py
num1 = int(input("Enter the 1st number: "))
num2 = int(input("Enter the 2nd number: "))

if num1 < num2:
    print("Smallest number is:",  num1)
elif num2 < num1:
    print("Smallest number is:",  num2)
else:
    print("Smallest number not found. Entered numbers are equal.")